### PR TITLE
Moving my local changes upstream

### PR DESCRIPTION
**Pull request by Kelvinrr**

### CODE REVIEW

File: ale/drivers/base.py [hunk 1]
```diff
@@ -369,109 +368,6 @@ def sun_position(self):
...
-
-    @property
-    def sun_position(self):
-        return {
-            "positions" : self._sun_position,
-            "velocities" : self._sun_velocity,
-            "unit" : "m"
-        }
-
-
```
Comment (jlaura) [inline ale/drivers/base.py]: Is all of this supposed to be pulled?

Comment (Kelvinrr) [reply ale/drivers/base.py]: I forgot to push some files that had merge conflicts dealt with.

File: ale/drivers/base.py [hunk 2]
```diff
@@ -318,7 +318,7 @@ def radii(self):
         return {
             "semimajor" : self._semimajor,
             "semiminor" : self._semiminor,
-            "unit" : "m" # default to meters
+            "unit" : "km" # default to meters
```
Comment (jlaura) [inline ale/drivers/base.py]: ? comment vs. unit

File: ale/drivers/base.py [hunk 3]
```diff
@@ -655,7 +659,7 @@ def _semimajor(self):
           Semimajor axis of the target body
         """
         rad = spice.bodvrd(self.target_name, 'RADII', 3)
-        return rad[1][1]
+        return rad[1][0]
```
Comment (jlaura) [inline ale/drivers/base.py]: Are these new changes?

Comment (Kelvinrr) [reply ale/drivers/base.py]: yeah, I seem to only get the correct semimajor/semininor this way. The docs for cspice are not clear about what the 3 element array holds, but when using CTX images 0th and 2nd element had the right radii.

Comment (jlaura) [reply ale/drivers/base.py]: :+1:

**Pull request merged**